.github/PREVIEW_ENVIRONMENTS.md

```
@@ -0,0 +1,58 @@
+
+- `CLOUDFLARE_ACCOUNT_ID` - Your Cloudflare account ID
+
+- `CLOUDFLARE_ZONE_ID` - Zone ID for `usestratum.dev`
+
+- `PREVIEW_KV_NAMESPACE_ID` - A shared KV namespace for preview environments
+  (Create once: `wrangler kv:namespace create "stratum-preview-state"`)
+
+## DNS Configuration
+
+Ensure you have a wildcard DNS record:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)